src/AppBundle/Repository/EventWikiRepository.php

```diff
@@ -91,10 +99,14 @@ public function getWikiFamilyName(string $value): ?string
     /**
      * Get the database name of the given (partial) domain name.
      * @param string $domain The domain name, without trailing '.org'.
-     * @return string Null if not found.
+     * @return string
```

Comment:
The phpcs didn't complain for whatever reason, but I think this should be `string|null`